doc/script_commands.txt

```diff
@@ -2612,6 +2612,79 @@ deleted.
 	// This would delete all elements of the array
 	deletearray(.@array);
 
+---------------------------------------
+//=====================================
+1.1 - Message formatting commands (tags)
```

Comment:
a bit off topic from this commit, but:
There is another tag, `<QUEST>` which exists in clients but I don't remember seeing it being used. I was unsure whether it was worth to create a `mesquest("label", questId)`

Reply:
I think the other tags might be interesting to have since they can open possibilities for custom scripts, but definitely at a lower priority if they're not used anywhere in official scripts (in a future, separate, PR)